pkg/network/tuntap/tap.go

```diff
@@ -0,0 +1,66 @@
+package tuntap
+
+import (
+	"fmt"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containernetworking/plugins/pkg/utils/sysctl"
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+const disableIPv6Template = "net.ipv6.conf.%s.disable_ipv6"
```

Comment:
I've seen this else where in the code. Let's extract it into the `ifaceutl` package so we can reuse it instead of copying the code.